pkg/client/gen.go

```diff
@@ -67,6 +80,23 @@ func (*SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 		}
 	}
 
+	plugins := []*manifest.Manifest{}
+	if includes(cfg.Config.PluginSelections, e2ePluginName) {
+		plugins = append(plugins, e2eManifest(cfg))
+	}
+	if includes(cfg.Config.PluginSelections, systemdLogsName) {
+		plugins = append(plugins, systemdLogsManifest(cfg))
+	}
+
+	pluginYAML := []string{}
+	for _, v := range plugins {
+		yaml, err := kuberuntime.Encode(manifest.Encoder, v)
+		if err != nil {
+			return nil, errors.Wrapf(err, "serializing plugin %v as YAML", v.SonobuoyConfig.PluginName)
+		}
+		pluginYAML = append(pluginYAML, strings.TrimSpace(string(yaml)))
```

Comment:
I was getting an EOL character or something that led to an extra, empty line in the template when rendered. Trimming it seems reasonable and fixed the issue.